### PR TITLE
bump encointer deps to polkadot-v1.13.0 and re-enable encointer

### PR DESCRIPTION
**Pull request by clangenb**
Ready to merge.

cc @brenzi

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2743,15 +2744,30 @@ dependencies = [
...
 ]
 
+[[package]]
+name = "crc"
+version = "2.1.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "49fc9a695bca7f35f5f4c15cddc84415f66a74ea78eef08e90c5024f2b540e23"
+dependencies = [
+ "crc-catalog 1.1.1",
+]
```
Comment (clangenb) [inline Cargo.lock:2754]: We will get rid of this old dep in the next encointer-release aka for polkadot-v1.13.0, see https://github.com/encointer/pallets/pull/396

**Comment by ggwpez** [comment]
Thanks! I a currently waiting for the 1.13 crate-io release from our release team, then i will merge this in.

**Comment by clangenb** [comment]
Yeah, the best way forward is probably:

1. You update your branch to v1.13.0
2. I rebase my branch and update to v1.13.0

Then we merge. Seems to be the most hassle-free to me.

**Comment by clangenb** [comment]
@ggwpez  Ready to merge IMO.

**Comment by ggwpez** [comment]
🚀

**Pull request merged**